entrypoint.sh

```diff
@@ -84,11 +84,9 @@ settingsConfiguration() {
         fi
 
         if [ "$setting_type" = "plain" ]; then
-            echo "$setting_key = $setting_var" >> /healthchecks/hc/settings.py
-        else
-            echo "$setting_key = \"$setting_var\"" >> /healthchecks/hc/settings.py
-        fi
-        echo "Added \"$setting_key\" (value \"$setting_var\") to settings.py"
+            echo "$setting_key = $setting_var" >> /healthchecks/hc/local_settings.py
+	fi
```

Comment:
The `else` is needed as otherwise only values from type `plain` are added to the config.
Please just change the target file to `local_settings.py` here.

Reply:
yeah, was just on it.
Sitting outside and enjoying the sun ... missed that at first.